### PR TITLE
Add allowInvisibleElements

### PR DESCRIPTION
**Pull request by rjaros87**
Fix for https://github.com/appium/appium-uiautomator2-server/issues/61

### CODE REVIEW

**Comment by rjaros87** [comment]
Correlated with https://github.com/appium/appium-uiautomator2-server/pull/63

File: lib/driver.js [hunk 1]
```diff
@@ -413,14 +413,9 @@ class AndroidUiautomator2Driver extends BaseDriver {
...
-  async setCompressedLayoutHierarchy (compress) {
-    await this.uiautomator2.jwproxy.command('/appium/device/compressedLayoutHierarchy', 'POST', {compressLayout: compress});
-  }
-
   async onSettingsUpdate (key, value) {
-    if (key === "ignoreUnimportantViews") {
-      await this.setCompressedLayoutHierarchy(value);
-    }
+    let settings = {[key]: value};
+    await this.uiautomator2.jwproxy.command('/appium/settings', 'POST', {settings: settings});
```
Comment (imurchie) [inline lib/driver.js]: `{settings: settings}` can just be `{settings}` (see https://github.com/getify/You-Dont-Know-JS/blob/master/es6%20%26%20beyond/ch2.md#concise-properties)

**Comment by sravanmedarapu** [comment]
It will be great if you can add test to verify this :)

**Comment by rjaros87** [comment]
@sravanmedarapu test already exists in https://github.com/appium/appium-uiautomator2-driver/blob/master/test/functional/commands/find/by-xpath-e2e-specs.js#L67 for `updateSettings` .
I've added a separate test for `allowInvisibleElemets` starting the Settings application because in the `ApiDemos` app I can't find for example the `statusBarBackground`. I'm not sure, but this might be because of a customized theme in the `ApiDemos`.

**Pull request merged**